packages/vaadin-themable-mixin/vaadin-themable-mixin.js

```diff
@@ -204,6 +204,11 @@ export const ThemableMixin = (superClass) =>
     static finalize() {
       super.finalize();
 
+      // Make sure not to run the logic intended for PolymerElement when LitElement is used.
+      if (this.prototype instanceof LitElement) {
```

Comment:
Importing the whole LitElement class for this would increase bundle size unnecessarily 😕 
Maybe we could instead check for `elementStyles` which is set in [`finalize()`](https://github.com/lit/lit/blob/4f1b923ad52f4eec462269080ad2226bb0d8c979/packages/reactive-element/src/reactive-element.ts#L773)?